tools/devops/automation/templates/build/publish-nugets.yml

```diff
@@ -4,16 +4,6 @@ steps:
 
 # do not publish on pull requets
 - ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
-  - task: NuGetCommand@2
-    displayName: 'Publish Nugets to xamarin-impl'
-    inputs:
-      command: push
-      packagesToPush: $(Build.SourcesDirectory)/package/*.nupkg
-      nuGetFeedType: external
-      publishFeedCredentials: xamarin-impl public feed
-    condition: and(succeeded(), eq(variables['configuration.BuildNugets'], 'True'))
-    continueOnError: true # should not stop the build since is not official just yet.
```

Comment:
Will this prevent dotnet/maui from consuming xamarin-macios builds?

I see where the other `nuget push` command is turned off for now.

Reply:
I mentioned this briefly above but its easy to lose comments in the build/test reporting. I had planned on testing this change last once the rest of the yaml and build changes were approved.